### PR TITLE
add upbound provider when using eksctl bootstrap

### LINKED ISSUE #93: [FEATURE] Add upbound provider support when using eksctl bootstrap
#### Is your feature request related to a problem? Please describe

Add upbound provider support when using eksctl bootstrap

#### Describe the solution you'd like

Easy way to try out both provider aws and upbound

### PR DESCRIPTION
**Pull request by csantanapr**
### What does this PR do?

add upbound provider when using eksctl bootstrap

### Motivation

Resolves #93 

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)

- [x] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR

- [x] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature

- [x] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [x] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by csantanapr** [comment]
@candonov PTAL when you have chance

File: bootstrap/eksctl/README.md [hunk 1]
```diff
@@ -99,32 +99,93 @@ See the [IRSA documentation](https://docs.aws.amazon.com/eks/latest/userguide/cr
...
 ```
 
 ## Install Crossplane
 
 ### Helm
 
-```bash
-kubectl create namespace crossplane-system
 
+#### (Option 1) Install Crossplane upstream helm chart
```
Comment (candonov) [inline bootstrap/eksctl/README.md:112]:
```suggestion
#### (Option 1) Install [Crossplane community helm chart](https://github.com/crossplane/crossplane/tree/master/cluster/charts/crossplane)
```

File: bootstrap/eksctl/README.md [hunk 2]
```diff
@@ -99,32 +99,93 @@ See the [IRSA documentation](https://docs.aws.amazon.com/eks/latest/userguide/cr
...
 helm repo update
 
-helm install crossplane --namespace crossplane-system --version 1.10.1 crossplane-stable/crossplane
+helm install crossplane crossplane-stable/crossplane /
+--namespace crossplane-system \
+--create-namespace \
+--version 1.10.2
 ```
 
+#### (Option 2) Install Crossplane using downstream Upbound
```
Comment (candonov) [inline bootstrap/eksctl/README.md:123]:
```suggestion
#### (Option 2) Install Crossplane using [Upbound Universal Crossplane (UXP) helm chart](https://github.com/upbound/universal-crossplane/tree/main/cluster/charts/universal-crossplane)
```

File: bootstrap/eksctl/README.md [hunk 3]
```diff
@@ -99,32 +99,93 @@ See the [IRSA documentation](https://docs.aws.amazon.com/eks/latest/userguide/cr
...
+
 ```bash
-# wait for the provider CRD to be ready.
+helm repo add upbound-stable https://charts.upbound.io/stable
+helm repo update
+
+helm install crossplane upbound-stable/universal-crossplane \
+--namespace crossplane-system \
+--create-namespace \
+--version 1.10.2-up.1
```
Comment (candonov) [inline bootstrap/eksctl/README.md:132]:
```suggestion
--version 1.10.2-up.1 # Get the latest version from https://github.com/upbound/universal-crossplane
```

File: bootstrap/eksctl/README.md [hunk 4]
```diff
@@ -99,32 +99,93 @@ See the [IRSA documentation](https://docs.aws.amazon.com/eks/latest/userguide/cr
...
+#### (Option 1) Install Crossplane upstream helm chart
+```bash
 helm repo add crossplane-stable https://charts.crossplane.io/stable
 helm repo update
 
-helm install crossplane --namespace crossplane-system --version 1.10.1 crossplane-stable/crossplane
+helm install crossplane crossplane-stable/crossplane /
+--namespace crossplane-system \
+--create-namespace \
+--version 1.10.2
```
Comment (candonov) [inline bootstrap/eksctl/README.md:120]:
```suggestion
--version 1.10.2 # Get the latest version from https://github.com/crossplane/crossplane
```

**Review by candonov — APPROVED** [review]
/lgtm

File: bootstrap/eksctl/eksctl.yaml [hunk 5]
```diff
@@ -6,10 +6,10 @@ kind: ClusterConfig
 metadata:
   name: crossplane-blueprints
   region: us-west-2
-  version: "1.23"
+  version: "1.24"
 managedNodeGroups:
   - name: managed-ng-1
-    instanceType: m5.large
+    instanceType: t3.small
```
Comment (nabuskey) [inline bootstrap/eksctl/eksctl.yaml:12]: This is not recommend. EMR on EKS example requires 1G memory and may not get scheduled with small overhead especially when users decide to install something on top to test. @csantanapr

Comment (csantanapr) [reply bootstrap/eksctl/eksctl.yaml:12]: EMR on EKS? We don't have any example on this repo that would not work with 3 t3.small nodes

For an user to get started to learn crossplane and aws they don't need to create large nodes, the can start simple for the minimum to be use as an EKS control plane.

What about if they want to run Kubeflow, or Kafka, or EMR, or Cassandra? They need to adjust the cluster size for the workload they are running. 

But this is your repo I can revert it to m5 @nabuskey

Comment (nabuskey) [reply bootstrap/eksctl/eksctl.yaml:12]: The composition we have in the repo for EMR on EKS will launch jobs into the same cluster so it's very possible pods may not be scheduled with a few daemon sets installed. I actually ran into it when I was writing the composition. Thanks for fixing it!
Also I don't consider this repository mine. We are all maintainers of this repo.

**Pull request merged**